src/rating-indicator.scss

```diff
@@ -0,0 +1,146 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./icon";
+
+$block: #{$fd-namespace}-rating-indicator;
+
+.#{$block} {
+  $unratedColor: var(--sapContent_UnratedColor);
```

Comment:
I think you forgot this one :)

Reply:
it used in multiple places in the code

Reply:
I was referring to the naming of the variable. Should be `$fd-rating-indicator-unrated-color`